src/cosmosdb-preview/azext_cosmosdb_preview/_params.py

```diff
@@ -307,6 +307,7 @@ def load_arguments(self, _):
         c.argument('tables_to_restore', nargs='+', action=CreateTableRestoreResource, is_preview=True)
         c.argument('assign_identity', nargs='*', help="Assign system or user assigned identities separated by spaces. Use '[system]' to refer system assigned identity.", is_preview=True)
         c.argument('default_identity', help="The primary identity to access key vault in CMK related features. e.g. 'FirstPartyIdentity', 'SystemAssignedIdentity' and more.", is_preview=True)
+        c.argument('source_backup_location', help="The source backup location for restore.", is_preview=True)
```

Comment:
We should update the "help" string for the 'location' property (line 304) since this source_backup_location can potentially overwrite that setting